DREAM3DReviewFilters/FindArrayStatistics.cpp

```diff
@@ -434,6 +462,74 @@ double findSummation(C<T, Ts...>& source)
   return sum;
 }
 
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+template <template <typename, typename...> class C, typename T, typename... Ts>
+std::vector<float> findHistogram(C<T, Ts...>& source, float histmin, float histmax, bool histfullrange, int32_t numBins)
+{
+
+  std::vector<float> Histogram(numBins, 0);
+  int overflow = 0;
+  int32_t bin = 0;
+  int32_t numPoints = source.size();
+
+  if(source.empty())
+  {
+    std::vector<float> empty(numBins, 0);
+    return empty;
+  }
+
+  float min = std::numeric_limits<float>::max();
+  float max = -1.0 * std::numeric_limits<float>::max();
```

Comment:
```suggestion
  float min = 0.0f;
  float max = 0.0f;
```
Since these variables will always be set in the next if-else block, they don't need to be set using std::numeric_limits::max().